docs/operation/lerobot_guide.rst

```diff
@@ -250,6 +251,43 @@ To upload intermediate checkpoints:
    $ huggingface-cli upload ${HF_USER}/act_aloha_test_${CKPT} \
       outputs/train/act_aloha_test/checkpoints/${CKPT}/pretrained_model
 
+Google Colab for Training
```

Comment:
Might be nice to link to Colab's FAQ: https://research.google.com/colaboratory/faq.html